### PR TITLE
Adds gasharpoon

### PR DESCRIPTION
**Pull request by vampirebat74**
what it says on the tin

## About The Pull Request

Adds a new "secret" realization engine reward and an interesting weapon option.

## Why It's Good For The Game

New coders might look at this and actually have decent ideas of what a realized suit should actually do

## Changelog
:cl:
add: Adds gasharpoon, pallid tuna, pedoq captain uniform
soundadd: added a new sound thingy
imageadd: added relevant icons for gasharpoon
/:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
>Touches 27 files

File: ModularTegustation/fishing/code/fish/LC13_fish/saltwater_fish.dm [hunk 1]
```diff
@@ -60,3 +60,20 @@
 		/datum/reagent/consumable/nutriment/organ_tissue = 1,
 		/datum/reagent/consumable/salt = 8,
 	)
+
```
Comment (Kitsunemitsu) [inline ModularTegustation/fishing/code/fish/LC13_fish/saltwater_fish.dm:63]: Going to request atomization for this, it's a small thing that can be merged almost instantly and doesn't really fit in this Gasharpoon PR.

File: code/modules/clothing/suits/ego_gear/non_abnormality/ucorp.dm [hunk 2]
```diff
@@ -0,0 +1,14 @@
+//Peqod Crew
+/obj/item/clothing/suit/armor/ego_gear/peqod_captain
```
Comment (Kitsunemitsu) [inline code/modules/clothing/suits/ego_gear/non_abnormality/ucorp.dm:2]: Same with this, it's a real quick atomization

File: code/modules/mob/living/simple_animal/abnormality/_tools/zayin/realizer.dm [hunk 3]
```diff
@@ -57,6 +57,7 @@
 		// Personal
 		/obj/item/managerbullet = /obj/item/clothing/suit/armor/ego_gear/realization/farmwatch,
 		/obj/item/storage/box/fireworks/dangerous = /obj/item/clothing/suit/armor/ego_gear/realization/spicebush,//from smuggler's satchels
+		/obj/item/food/fish/salt_water/tuna_pallid = /obj/item/clothing/suit/armor/ego_gear/realization/gasharpoon,
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/_tools/zayin/realizer.dm:60]: Ah, the tuna makes sense now. Still will request an atomization that I will merge like ASAP.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Small atomization requests just to make merging easier.

**Comment by vampirebat74** [comment]
> Small atomization requests just to make merging easier.

As requested, the files regarding the status effect, fish, fixer suit, and realization engine entry to  #2059. It will conflict due to uncommenting for the weapon but that's no big deal.

**Review by Kitsunemitsu — APPROVED** [review]
No massive outstanding issues

**Pull request merged**